docs/sample_inventories/scram_kraft.yml

```diff
@@ -0,0 +1,55 @@
+---
+##
+## The following is an example inventory file of the configuration required for setting up Confluent Platform with Kraft Controller with SCRAM.
+## You can use any of the below approaches to setup SCRAM communication with Kraft. Same can be followed for migration as well.
+## Option 1: This will set up controller-controller and controller-broker communication via plain and other communication via scram.
+## Option 2: This will set up controller-controller via plain and other communication via scram.
+
+all:
+  vars:
+    ansible_connection: ssh
+    ansible_user: ec2-user
+    ansible_become: true
+    ansible_ssh_private_key_file: /home/ec2-user/guest.pem
+    ansible_python_interpreter: /usr/bin/python3
+
+    ssl_enabled: true
+    sasl_protocol: scram
+
+    ## Option 1
+    kafka_controller_sasl_protocol: plain,scram
+
+kafka_controller:
+  vars:
+    ## Option 2
+    kafka_controller_sasl_protocol: plain,scram
```

Comment:
please create 2 separate inventory files for these cases instead of adding 2 options here. I think we can also ignore this one and recommend adding in `all` for simplicity

Reply:
done